tests/testthat/test_stat_density_ridges.R

```diff
@@ -110,3 +110,23 @@ test_that("alternative quantile function can be provided", {
   expect_setequal(out$datatype, c("ridgeline", "vline"))
   expect_equal(out$x[out$datatype=="vline"], mean(df$x))
 })
+
+test_that("unweighted densities are calculated correctly", {
+  df <- data.frame(x = rnorm(100),wts = runif(100))
+  df$wts <- df$wts / sum(df$wts)
+
+  gg_no_wts <- layer_data(ggplot(df, aes(x = x, y = 0)) + stat_density_ridges())
+  d_no_wts <- stats::density(df$x)
+
+  expect_equal(gg_no_wts$density,d_no_wts$y)
```

Comment:
Please always put a space after a comma (there are a few more cases like that in the following lines).